### DISCUSSION TITLE
Spring / Spring Boot Version Compatibility

### DISCUSSION THREAD

**Discussion by v3i1y** [Netflix/dgs-framework#433] [Q&A] [answered]
The documentation only mentioned that this framework needs Kotlin 1.4 compatibility-wise and nothing about Spring versions. Is this library backward compatible with older Spring Boot builds like 2.1?

**Comment by paulbakker** [comment, marked as answer]
Probably not. 2.3 is currently the minimum version we test with.
It might work, but we don't plan to specifically support older versions.

We should probably set some minimum requirements in the docs.
